vowpalwabbit/core/src/parse_args.cc

```diff
@@ -1110,6 +1111,7 @@ void parse_example_tweaks(options_i& options, VW::workspace& all)
                .help("Turn this on to disregard order in which features have been defined. This will lead to smaller "
                      "cache sizes"))
       .add(make_option("loss_function", loss_function)
+               .keep()
```

Comment:
There must be a reason why we have not specified keep for loss_function before this. So I am hesitant on this one

Reply:
I'll follow up with marco